client/src/pages/organizations/Form.js

```diff
@@ -344,6 +360,19 @@ class BaseOrganizationForm extends Component {
 		arrayHelpers.push({name: '', approvers: []})
 	}
 
+	removeApprovalStep = (arrayHelpers, index, step) => {
+		return API.query(/* GraphQL */`
+			approvalStepInUse(uuid:"${step.uuid}") 
+		`).then(data => {
+			if (data) {
+				this.setState({showRemoveApprovalStepAlert: true})
+			}
+			else {
```

Comment:
Our Javascript style has _else_ on the same line as the closing brace of the _if_: `} else {`